.github/workflows/ci.yml

```
@@ -147,3 +147,47 @@ jobs:
         continue-on-error: ${{ matrix.allowed_failure || false }}
         env: ${{ matrix.env }}
         run: bash ./Tools/ci-run.sh
+        
+
+  code_quality:
+    strategy:
+      # Allows for matrix sub-jobs to fail without canceling the rest
+      fail-fast: false
+    runs-on: ubuntu-18.04
+
+    env:
+      BACKEND: c
```

The C++ part of the test suite is mostly a superset of the C tests.
```suggestion
      BACKEND: cpp
```